deployment/Dockerfile.prod

```diff
@@ -0,0 +1,16 @@
+FROM node:12-alpine
+
+RUN mkdir -p /usr/app
+WORKDIR /usr/app
+
+COPY . .
```

Comment:
I don't think we should copy everything in the prod image, cause we might end up copying files that are at best irrelevant and at worst that should not be accessible by the end user (like test files, or our own .env.prod file for example). I know that for now we deploy this image on AWS and we're supposed to be the only ones accessing it, but if one day we want to push this image to the Docker Hub for instance then currently we can't. I'd recommend sticking to listing the files and folders that are actually sources we want to copy.

Reply:
Yes i think it's safer to target manually each files/folders instead of copying everything.